chain_info/Shibuya.yaml

```diff
@@ -0,0 +1,46 @@
+name: Shiden

```

Comment:
Name should be the same as the file name. In this case, `Shibuya`.

Reply:
Thx.